common/rfb/EncodeManager.cxx

```diff
@@ -77,6 +82,12 @@ struct RectInfo {
   Palette palette;
 };
 
+struct QualityInfo {
+  struct timeval lastUpdate;
+  Rect rect;
```

Comment:
I requested individual rect because I was afraid that a small GIF animation on a webpage would result in noticeably bad quality in static neighboring portions.